src/Symfony/Component/HttpKernel/EventListener/AbstractSessionListener.php

```diff
@@ -40,11 +41,13 @@ abstract class AbstractSessionListener implements EventSubscriberInterface
     const NO_AUTO_CACHE_CONTROL_HEADER = 'Symfony-Session-NoAutoCacheControl';
 
     protected $container;
+    private $debug;
     private $sessionUsageStack = [];
 
-    public function __construct(ContainerInterface $container = null)
+    public function __construct(ContainerInterface $container = null, bool $debug = true)
```

Comment:
`$debug = true` does not sound like a safe default.

Reply:
Yes indeed!